compiler/luci/export/src/CircleTensorExporter.cpp

```diff
@@ -562,6 +562,8 @@ bool has_same_values(luci::CircleConst *lhs, luci::CircleConst *rhs)
       return has_same_elements<loco::DataType::S4>(lhs, rhs);
 
     case loco::DataType::S8:
+      if (lhs->size<loco::DataType::S8>() != rhs->size<loco::DataType::S8>())
+        return false;
```

Comment:
Q) this assumes only S8,U8 supports compression for current implementation.
Am I correct?
- can you point me if there is any comments about this?